variants/STM32L0xx/L011K(3-4)U_L021K4U/variant_NUCLEO_L011K4.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+#if defined(ARDUINO_NUCLEO_L011K4)
+#include "pins_arduino.h"
+
+// Digital PinName array
+const PinName digitalPin[] = {
+  PA_0,   // D0/A0
+  PA_1,   // D1/A1
+  PA_2,   // D2/A2
+  PA_3,   // D3/A3
+  PA_4,   // D4/A4
+  PA_5,   // D5/A5
+  PA_6,   // D6/A6
+  PA_7,   // D7/A7
+  PA_8,   // D8
+  PA_9,   // D9
+  PA_10,  // D10
+  PA_11,  // D11
+  PA_12,  // D12
+  PA_13,  // D13
+  PA_14,  // D14
+  PA_15,  // D15
+  PB_0,   // D16/A8
+  PB_1,   // D17/A9
+  PB_2,   // D18
+  PB_3,   // D19
+  PB_4,   // D20
+  PB_5,   // D21
+  PB_6,   // D22
+  PB_7,   // D23
+  PB_8,   // D24
+  PB_9,   // D25
+  PC_14,  // D26
+  PC_15   // D27
```

Comment:
Should be ordered following the connectors

Reply:
Please can you clarify so that i can get this correct in the future. Should the be ordered as per the physical order on the dev board, or the MCU?

If the pins are ordered by the dev board pn locations, i.e. 
![image](https://user-images.githubusercontent.com/35117191/187890011-4a612dcb-46ea-42d6-994a-07a7f3ad6852.png)
Is the pin ordering to be done similar to "DIP" i.e. 
![image](https://user-images.githubusercontent.com/35117191/187890171-0fbefac2-c150-4950-b2cc-4d9f8bd4826f.png)

Reply:
In the case of existing boards, we used the physical order. For generic, pins are simply listed per GPIO port: PAx, PBx,...

Reply:
So just to confirm, are you happy with reading the physical order as clockwise? as per the 'DIP' example above.